openedx/features/badging/tests/test_helpers.py

```diff
@@ -144,14 +160,14 @@ def test_get_course_badges_successfully(self, mock_get_course_by_id,
         self.assertIn('conversationalist', badge_keys)
         self.assertNotIn('team', badge_keys)
 
-    @mock.patch('openedx.features.badging.helpers.add_badge_earned_date')
-    @mock.patch('openedx.features.badging.helpers.filter_earned_badge_by_joined_team')
-    @mock.patch('openedx.features.badging.helpers.is_teams_feature_enabled')
-    @mock.patch('openedx.features.badging.helpers.get_course_by_id')
+    @mock.patch('openedx.features.badging.helpers.badges.add_badge_earned_date')
+    @mock.patch('openedx.features.badging.helpers.badges.filter_earned_badge_by_joined_team')
+    @mock.patch('openedx.features.badging.helpers.badges.is_teams_feature_enabled')
+    @mock.patch('openedx.features.badging.helpers.badges.get_course_by_id')
     def test_get_course_badges_user_not_joined_any_course_team(self, mock_get_course_by_id,
-                                                           mock_is_teams_feature_enabled,
-                                                           mock_filter_earned_badge_by_joined_team,
-                                                           mock_add_badge_earned_date):
+                                                       mock_is_teams_feature_enabled,
+                                                       mock_filter_earned_badge_by_joined_team,
+                                                       mock_add_badge_earned_date):  # pylint: disable=unused-argument
```

Comment:
Please update according to [edx recommendations](https://edx.readthedocs.io/projects/edx-developer-guide/en/latest/style_guides/python-guidelines.html#breaking-long-lines).